app/client/run_dev_client.sh

```diff
@@ -4,7 +4,6 @@ SCHEDULER_URL=$(minikube service scheduler --url)
 MCMC_STATS_SERVER_URL=$(minikube service mcmc-stats-server --url)
 GRAPHITE_URL=$(minikube service graphite --url | head -n1)
 
-export SCHEDULER_URL=$SCHEDULER_URL
-export GRAPHITE_URL=$GRAPHITE_URL
-export MCMC_STATS_URL=$MCMC_STATS_SERVER_URL
+export SCHEDULER_URL MCMC_STATS_SERVER_URL GRAPHITE_URL
```

Comment:
We don't want to export `MCMC_STATS_SERVER_URL` here but rather `MCMC_STATS_URL`. Compare with 99269cd3769f0da5d346286a23bea98f09bdeb9a. Having multiple variables exported on a single line or multiple lines is a matter of taste. I prefer multiple lines because the lines can easily be sorted and the git diffs are a bit cleaner (say if a variable gets deleted).